conversions/decimal_to_binary.py

```diff
@@ -1,7 +1,7 @@
 """Convert a Decimal Number to a Binary Number."""
 
 
-def decimal_to_binary(num):
+def decimal_to_binary(num: int) -> str:
```

Comment:
It might be even cooler to actually return a binary number. i.e. 0b123 instead of "0b123".